README.md

```diff
@@ -431,9 +431,15 @@ while not pointer is None:
 ```python3
 from instagram import WebAgent
 
-settings = {"proxies": {"any_ip": any_port}}
+settings = {
+    "proxies": {
+        'http': 'http://example.net:8888',
+        'https': 'https://example.net:8888'
+    }
+}
```

Comment:
Is this description fit settings for "requests" and for "aiohttp"?

Reply:
No, sorry, but I've tested it only with "requests". Maybe your config works with "aiohttp".

Reply:
```suggestion
agent = WebAgent(settings=settings)
```

But this string cause error anyway.

Reply:
Thank you! If this work for requests - it is corect